packages/core/src/base-chart.ts

```diff
@@ -162,6 +164,16 @@ export class BaseChart {
 			}
 		});
 
+		if (typeof this.fixedDataLabels === "undefined") {
```

Comment:
I'd either flip this if statement around, or do `if(!this.fixedDataLabels)`, unless we need to check for specifically "undefined".